.github/workflows/ci.yml

```
@@ -0,0 +1,157 @@
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)